packages/page-mixer/src/components/deposit/DepositConsole.tsx

```diff
@@ -1,36 +1,25 @@
-import { BalanceInputValue, eliminateGap } from '@webb-dapp/react-components';
+import { BalanceInputValue } from '@webb-dapp/react-components';
 import AmountInput from '@webb-dapp/react-components/AmountInput/AmountInput';
 import { TokenInput } from '@webb-dapp/react-components/TokenInput';
-import { useApi, useBalance, useBalanceValidator, useConstants } from '@webb-dapp/react-hooks';
+import { useApi, useConstants, useMixer } from '@webb-dapp/react-hooks';
 import { useInputValue } from '@webb-dapp/react-hooks/useInputValue';
 import { Col, Row, SpaceBox } from '@webb-dapp/ui-components';
-import { FixedPointNumber } from '@webb-tools/sdk-core';
+import { Token, token2CurrencyId } from '@webb-tools/sdk-core';
 import { CurrencyId } from '@webb-tools/types/interfaces';
 import React, { FC, useCallback, useMemo, useState } from 'react';
 
 import { CardRoot, CardSubTitle, CardTitle, CTxButton, DepositTitle } from '../common';
 
 export const DepositConsole: FC = () => {
   const { api } = useApi();
-  const [token, setToken, { error: tokenError, setValidator: setTokenValidator }] = useInputValue<BalanceInputValue>({
+  const { mixerInfos } = useMixer();
```

Comment:
I can see an issue with the current structure for `useMixer` it should be used inside the provider for the mixer-context, not directly.

we may move the code from the `useMixer` to the provider directly, or continue with the current and instead call `useMixerProvider`.



```suggestion
  const  mixerInfos  = useMixerInfos();
```